gotagger.go

```diff
@@ -75,6 +82,55 @@ type Config struct {
 	*/
 }
 
+// ReadCommitTypeMappings reads a config file containing mappings of commit type to semver increment. Mappings determine
+// how much to increment the semver based on the commit type. Unknown commit types will fall back to the config default.
+// Invalid increments will throw an error. Duplicate type definitions will take the last entry.
+func (c *Config) ReadCommitTypeMappings() error {
```

Comment:
Let's refactor this to `ParseJSON([]byte) error`, and drop the ConfigFile member.

Embed the fields from IncrementConfig in Config.

Reply:
Done